config/environments/test.rb

```diff
@@ -39,7 +39,4 @@
 
   # Raises error for missing translations
   # config.action_view.raise_on_missing_translations = true
-
-  config.neo4j.session_type = :server_db
-  config.neo4j.session_path = 'http://localhost:7475'
```

Comment:
Ah, interesting!  I hadn't heard of `figaro` before but that seems like that would do it.

I've used the `dotenv` gem previously with good luck.  I use it in this asset portal example app:

https://github.com/neo4j-examples/asset_portal

Specifically there I'm using `dotenv-rails`.  I'm just seeing the comparison now on the `figaro` README and it doesn't seem like there's much difference aside from `figaro` being Rails focused (aside from the `dotenv-rails` gem, I suppose) and the different ways of making configuration files.